code/math/curve.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) Volition, Inc. 1999.  All rights reserved.
+ *
+ * All source code herein is the property of Volition, Inc. You may not sell 
+ * or otherwise commercially exploit the source or things you created based on the 
+ * source.
+ *
+*/ 
```

Comment:
Do we need to have this?  There's no volition source code in this file or curve.cpp? Also, SCP contributions are under the Unlicense.  I don't think I've seen anyone else add this to other new files.

Reply:
Idk I specifically looked at other added files and all the ones I could find had it, which is the only reason I did, but now I can't find any which do so I guess maybe not.

Reply:
Some brand new files have a variation on it, for example objectdock.cpp or mod_table.cpp.  But that's more of a "monkey see, monkey do" than a rule.